release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -79,7 +79,191 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
+* Compute nodes must run {op-system-first}
+* Persistent shared storage must be provisioned by using either {rh-storage-first} or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA
+
+[id="ocp-4-12-ibm-power"]
+=== {ibmpowerProductName}
+
+With this release, {ibmpowerProductName} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on {ibmpowerProductName} with {product-title} {product-version}:
+
+* Assisted Installer
+* Cluster Resource Override Operator
+* Egress IP
+* Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
+* MetalLB Operator
+* Network-Bound Disk Encryption - External Tang Server
+
+[discrete]
+==== Supported features
+
+The following features are also supported on {ibmpowerProductName}:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** File Integrity Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** SR-IOV Network Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+* The following Multus CNI plugins are supported:
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+* Alternate authentication providers
+* Cloud controller manager for IBM Cloud
+* Cron jobs
+* CSI Volumes
+** Cloning
+** Expansion
+** Snapshot
+* Descheduler
+* Encrypting data stored in etcd
+* Helm
+* Horizontal pod autoscaling
+* IPv6
+* Monitoring for user-defined projects
+* Multipathing
+* Multus SR-IOV
+* Operator API
+* OC CLI plugins
+* OVN-Kubernetes, including IPsec encryption
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* PodDisruptionBudget
+* Scheduler profiles
+* Stream Control Transmission Protocol (SCTP)
+* Support for multiple network interfaces
+* Support for Power10
+* Three-node cluster support
+* Topology Manager
+* 4K Disk Support
+
+[discrete]
+==== Restrictions
+
+The following restrictions impact {product-title} on {ibmpowerProductName}:
+
+* Automatic repair of damaged machines with machine health checking
+* {openshift-local-productname}
+* Controlling overcommit and managing container density on nodes
+* OpenShift Metering
```

Comment:
```suggestion

```
Support for Metering has been removed. We don't need to call out the restriction anymore.